src/hist/plot.py

```diff
@@ -651,7 +658,9 @@ def _plot_ratiolike(
             )
 
         elif view == "pull":
-            pulls = (hist_values - compare_values) / hist_values_uncert
+            pulls: np.typing.NDArray[Any] = (
```

Comment:
We should check and see why this requires a type now - one of the three input values here (hist_values, compare_values, hist_values_uncert) might be missing a type. Don't have to happen in this PR though.

Reply:
From @amangoel185:

`reveal_type` on the values gives:

```
src/hist/plot.py:661: note: Revealed type is "numpy.ndarray[Any, Any]" # hist_values
src/hist/plot.py:662: note: Revealed type is "numpy.ndarray[Any, numpy.dtype[Any]]" # compare_values
src/hist/plot.py:663: note: Revealed type is "numpy.ndarray[Any, numpy.dtype[Any]]" # hist_values_uncert
```